app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    comment = Comment.new(comment_params)
+    respond_to do |res|
+      res.html do
+        comment.save
+        redirect_to user_posts_path(current_user)
+      end
+    end
+  end
+
+  def comment_params
+    params
+      .require(:comment)
+      .permit(:text)
+      .merge(author: current_user, post_id: params.require(:post_id))
+  end
```

Comment:
- [ ] OPTIONAL: To follow SOLID principles, I recommend you to consider making comment_params a private method, since you don't need to access this method from outside the class. 
This refers to other controllers as well 👍🏻